manual/src/cmds/native-debugger.etex

```diff
@@ -0,0 +1,289 @@
+\chapter{Native debugger} \label{c:native-debugger}
+%HEVEA\cutname{native-debugger.html}
+
+\section{s:native-debugger-preliminaries}{Preliminaries}
+
+This chapter describes the support for debugging executables built with \texttt{ocamlopt}, the native-code compiler, using GDB or LLDB on Linux, macOS, and FreeBSD. We will call this \texttt{native debugging}, in contrastto bytecode debugging supported via \texttt{ocamldebug} (see chapter~\ref{c:debugger}).
+
+\subsection{ss:native-debugger-dwarf}{DWARF}
+
+The OCaml compiler uses the \href{http://dwarfstd.org/}{DWARF} debugging information file format to describe the debug information it generates. DWARF is a debugging information file format used by many compilers and debuggers to support source level debugging, and it is used by Linux ELF, macOS Mach-O and FreeBSD ELF.
```

Comment:
```suggestion
The OCaml compiler uses the \href{http://dwarfstd.org/}{DWARF} to describe the debug information it generates. DWARF is a debugging information file format widely used by compilers and debuggers to support source-level debugging across Linux ELF, macOS Mach-O, and FreeBSD ELF systems.

```
Since you define DWARF in the second sentence, it probably doesn't need to be in the first where it complicates the structure. Also I tightened up the second sentence somewhat. 

Alternatively :: 

The OCaml compiler uses \href{http://dwarfstd.org/}{DWARF} debugging information file format to describe the debug information it generates. DWARF is widely used by compilers and debuggers to support source-level debugging across Linux ELF, macOS Mach-O, and FreeBSD ELF systems.